test/suites/auth.sh

```diff
@@ -95,7 +117,21 @@ effective_groups:
 effective_permissions: []
 EOF
 )
-  lxc auth identity info oidc: | grep -Fz "${expected}"
+  lxc auth identity info oidc: | grep -Fz "${expectedOIDCInfo}"
```

Comment:
I went to the man page for `grep -z` ;)
Still not sure why a `null` char is wanted here.

Reply:
**If** the full command output is supposed to match the `${expectedOIDCInfo}` content, this could be done in a simple comparison `[ "$(lxc auth identity info oidc:)" = "${expectedOIDCInfo}" ]` as that works on multi-line variables too.

Reply:
grep -z was used so that it would match across multiple lines. I've changed to use the plain `=` test.

Reply:
I didn't know that `grep -z` trick, thanks!